apps/root/src/common/mocks/earn.ts

```diff
@@ -207,6 +208,7 @@ export const sdkDetailedEarnPositionMock: DetailedSdkEarnPosition = {
         hash: '0xhash',
         timestamp: 1720042607,
       },
+      assetPrice: 1,
```

Comment:
does this actually come in the sdk call?

Reply:
Yes, I forgot to add it as a comment. I have validated it with our CTO and we are okay :D